helmfile/01-applications.yaml

```diff
@@ -74,7 +74,11 @@ releases:
   chart: ingress-nginx/ingress-nginx
   version: 2.10.0
   missingFileHandler: Error
-  wait: true
+{{ if eq .Environment.Name "service_cluster" }}
+  needs:
+  - monitoring/user-grafana
+  - dex/dex
```

Comment:
This seems really weird and wrong imo.

Reply:
@jakubkrzywda I assume the intention here is that nothing happens in the cluster between the time the Webhook is installed and the Pod handling the Webhook is Ready. I assume that "needs" is not really used to signal a dependency, rather a "non-concurrency"/serialization among releases.

Could we then invert this dependency? I.e., user-grafana and dex needs a fully working ingres-nginx and not the other way around? This would feel a bit more "logical" to me.

Reply:
@cristiklein I tried the invert of this and I didn't manage to make it work that way.

Maybe what should be done is to investigate more the exact cause of the error before proposing the solution.

The solution here is the only one I found working for my specific setup with single node clusters. All the other approaches were leading to the error with `failed calling webhook`.